examples/src/components/ExecuteExample.jsx

```diff
@@ -0,0 +1,100 @@
+            executionNumber: 0,
+            willFail: true
+        };
+        this.onLoadingChanged = this.onLoadingChanged.bind(this);
+        this.onError = this.onError.bind(this);
+        this.retry = this.retry.bind(this);
+    }
+
+    onLoadingChanged({ isLoading }) {
+        // eslint-disable-next-line no-console
+        console.log('isLoading', isLoading);
+        // onLoadingChanged must reset error, so that we are not in error during loading
+        // onError is run after onLoadingChanged, so we do not have to worry about overriding current error
+        this.setState({
+            isLoading,
+            error: null
+        });
+    }
+
+    onError(error) {
+        // eslint-disable-next-line no-console
+        console.log('onError', error);
+        this.setState({
+            error
+        });
+    }
+
+    retry() {
+        // eslint-disable-next-line no-console
+        console.log('retry');
+        // We need to track executionNumber so that we can remount Execute component
+        // In order to showcase error states, here we also decide if the next execution will fail or not
+        this.setState({
+            executionNumber: this.state.executionNumber + 1,
+            willFail: this.state.executionNumber % 2
+        });
+    }
+
+    render() {
+        const { error, isLoading, executionNumber, willFail } = this.state;
+        const projectId = 'hdint8y1xvk9oyns1r0iefknwivzlb4g';
+        const afm = {
+            measures: [
+                {
+                    localIdentifier: 'measure',
+                    definition: {
+                        measure: {
+                            item: {
+                                // In order to showcase the fail state, we send invalid measure uri
+                                uri: willFail ? `/gdc/md/${projectId}/obj/15415` : null
+                            },
+                            aggregation: 'sum'
+                        }
+                    }
+                }
+            ]
+        };
+
+        let status = null;
+
+        // This is how we render loading and error states
+        if (isLoading) {
+            status = <p>Loading…</p>;
+        } else if (error) {
+            status = <p>Oops, simulated error! Retry?</p>;
+        }
+
+        return (<div>
+            {status}
+            <p><button onClick={this.retry}>Retry</button> (fails every other retry attempt)</p>
+            {/* We need to render the Execute component even during loading otherwise the ongoing request is cancelled */}
+            <Execute
+                key={executionNumber}
+                afm={afm}
+                projectId={projectId}
+                onLoadingChanged={this.onLoadingChanged}
+                onError={this.onError}
+            >{(executionResult) => {
+                    return !error ? (<div>
+                        <h3>Example KPI: <big>{executionResult.result.executionResult.executionResult.data[0]}</big></h3>
```

Comment:
Its confusing to call it KPI when not using the component - what about `Execution value`
and below add something like `Full execution response and result as JSON:`